app/packages/core/src/plugins/SchemaIO/components/GridView.tsx

```diff
@@ -18,14 +18,20 @@ export default function GridView(props: ViewPropsType) {
   }
 
   const layoutHeight = props?.layout?.height;
+  const parsedGap = parseGap(gap);
+  const adjustedLayoutWidth = getAdjustedLayoutWidth(
+    props?.layout?.width,
+    parsedGap
+  );
 
   const baseGridProps: BoxProps = {
     sx: {
-      display: "grid",
-      gap,
+      display: "flex",
```

Comment:
big change... how was this tested?

Reply:
Tested with the existing panels and ensuring they look same as before these changes with better auto-flow